stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/MessageComposerTheme.kt

```diff
@@ -213,3 +216,62 @@ public data class ComposerInputFieldTheme(
         }
     }
 }
+
+/**
+ * Defines the theming options for the different composer actions.
+ *
+ * @param attachmentsButton The style for the attachments button.
+ * @param commandsButton The style for the commands button.
+ * @param sendButton The style for the send button.
+ */
+public data class ComposerActionsTheme(
```

Comment:
We can attach `@Immuable` annotation here to make sure the properties of this theme style wouldn't be changed after the initial creation.